<launch>
  <node pkg="roslink" name="udp_client_roslink_bridge_ardrone_node" type="roslink_bridge_ardrone.py" output="screen"/>
  <param name="ground_station_ip" value='192.168.1.3' />
  <param name="ground_station_port" value='25500' />

  <param name="heartbeat_msg_rate" value="1" />
  <param name="robot_status_msg_rate" value="0.5" />
  <param name="global_motion_msg_rate" value="0.5" />
  <param name="gps_raw_info_msg_rate" value="0.5" />
  <param name="range_finder_data_msg_rate" value="0.5" />
  <param name="ROSLing_Image_msg_rate" value="1" />

  <param name="roslink_version" value="1" />
  <param name="ros_version" value="8" />
  <param name="system_id" value='12' />
  <param name="robot_name" value="MyArdrone2" />
  <param name="type" value="0" />
  <param name="owner_id" value="1" />
  <param name="key" value="N12223Y" />

</launch>

